foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/CountAction.java

```diff
@@ -77,7 +77,7 @@ public ActionResponse execute(CountRequest parameter) throws QueryStoreException
         try {
             if (parameter.isDistinct()){
                 SearchRequestBuilder query = getConnection().getClient()
-                        .prepareSearch(ElasticsearchUtils.getIndices(parameter.getTable()))
+                        .prepareSearch(ElasticsearchUtils.getIndices(parameter.getTable(), parameter))
```

Comment:
Might be in the future... Lets keep the options open for now.. At most, we might subclass this into a singleTableAction or something, but i didn't really see the value in doing the later for now. The former would make life a little simpler, but as i said, you never know..